packages/docs-reanimated/docs/scroll/scrollTo.mdx

```diff
@@ -72,6 +72,7 @@ import ScrollToSrc from '!!raw-loader!@site/src/examples/ScrollTo';
 ## Remarks
 
 - The `scrollTo` function can only be called from [the UI thread](/docs/fundamentals/glossary#ui-thread).
+- The usage with the `Animated.FlatList` component is the same as with the `Animated.ScrollView` component.
```

Comment:
I'd add it in remarks that it will usually work with other Scroll-like (of FlatList-like) components if they have a scrollView nested somewhere and are made animated components